src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,56 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final int OPERATION_COLUMN = 0;
+    private static final int AMOUNT_COLUMN = 1;
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String REGEX = ",";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        int supply = 0;
+        int buy = 0;
+        String[] strings = reedToStringArray(fromFileName);
+        for (String string : strings) {
+            String[] split = string.split(REGEX);
+            if (split[OPERATION_COLUMN].equals(SUPPLY)) {
+                supply += Integer.parseInt(split[AMOUNT_COLUMN]);
+            }
+            if (split[OPERATION_COLUMN].equals(BUY)) {
+                buy += Integer.parseInt(split[AMOUNT_COLUMN]);
+            }
+        }
+        writeToFile(toFileName, supply, buy);
+    }
+
+    private String[] reedToStringArray(String fromFileName) {
+        File file = new File(fromFileName);
+        List<String> list;
+        try {
+            list = Files.readAllLines(file.toPath());
+        } catch (Exception e) {
```

Comment:
why Exception, and not IOException? we should aim to catch the narrowest exception